controllers/news.controller.js

```diff
@@ -0,0 +1,16 @@
+const fetch = require('node-fetch');
+
+exports.getNews = async (req, res, next) => {
+  const source = `https://www.gamespot.com/api/games/?api_key=${process.env.GAMESPOTAPIKEY}&format=json`
+
+  try {
+    const response = await fetch(source);
+    const results = await response.json();
+
+    res.locals.news = results.results;
```

Comment:
I misspoke, in this context we should name the controllers and variables as articles.